.github/workflows/neon_extra_builds.yml

```diff
@@ -109,11 +117,6 @@ jobs:
           submodules: true
           fetch-depth: 1
 
-      - name: Install Ubuntu postgres dependencies
-        run: |
-          sudo apt update
-          sudo apt install build-essential libreadline-dev zlib1g-dev flex bison libseccomp-dev libssl-dev protobuf-compiler
-
```

Comment:
Are these already installed on gen3 or why remove these...?

Reply:
Before this PR, the "clean build" was run on GH Actions' runner, `ubuntu-latest`.
But now we need to write to our S3 bucket and that requires creds and tools installed, easiest way is to use something with all this, such as our runner, now gen3.

And yes, that has all tools installed, so no need to install them again.
I also have to disable cachepot now due to this runner switch.